nemo_skills/inference/server/reward_model.py

```diff
@@ -105,22 +105,21 @@ def __init__(self, **kwargs):
         model_list = self.oai_client.models.list()
         self.model = model_list.data[0].id
 
+    def _score_single_prompt(self, prompt, oai_client, model):
```

Comment:
client and model don't need to be passed, you can just access them with `self.` ?

Reply:
yes, self is also good